roles/confluent.kafka_broker/templates/kafka_server_jaas.conf.j2

```diff
@@ -1,17 +1,19 @@
-// Specifies a unique keytab and principal name for each broker
```

Comment:
this if statement is a bug, will need the kafkaserver block if kafka listeners have kerberos on them

question- is the KafkaServer block needed if just zookeeper sasl on?